### ISSUE TITLE
[tx] call to fatal() causes a segmentation fault

### ISSUE THREAD

**Issue by cjchapman** [adobe-type-tools/afdko#445]
In #444, the `tx` output (on OS X) ended with:

```
tx: (cfr) bad INDEX offset array
Segmentation fault: 11
```

...which was due to this call within `readCharStringsINDEX()` in `cffread.c`:

```C
                fatal(h, cfrErrINDEXOffset);
```

...and, as @blueshade7 pointed out in #444, "tx shouldn’t crash when fatal is called. It should gracefully exit."

**Comment by cjchapman** [comment]
There are several implementations of `fatal()` in several files that call `RAISE()` which then calls `os_raise()` which then calls `longjmp()` which triggers a segmentation fault.

I think the `fatal()` functions could probably just call `exit()` directly.

**Comment by miguelsousa** [comment]
Fixed by #495
